test/integration/transaction.spec.ts

```diff
@@ -1517,10 +1517,51 @@ describe('liquidjs-lib (transactions with psbt)', () => {
     psbt.finalizeAllInputs();
 
     const tx = psbt.extractTransaction();
-
     // build and broadcast to the Bitcoin RegTest network
     await regtestUtils.broadcast(tx.toHex());
   });
+
+  it('can create (and broadcast via 3PBP) an issuance transaction (unblinded)', async () => {
```

Comment:
The integration test does not pass: the `broadcast` function returns error 400 corresponding to a **sendrawtransaction RPC error: {"code":-26,"message":"bad-txns-in-ne-out, value in != value out (code 16)"}**.
However, the tx sent seems to be correct: 
`020000000001b9a16e79df8d60a90b7f3f55659219bbafc30c7d9c262d16cd93f37f7fbcf82b000000806a4730440220430fcc6ec2e120c1cc2d3a2c590597ade39bfa1f86a11daaa4f3d2e9af405c21022024735d5616313137a1329ab66e29e1979f591e612a1ca3ec2bc150fc92b350d601210328e0963aa67e1d4db8bcbafca157a27e26f8dd763ed7b845dce27dbcac82af96ffffffff000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000002540be400010000000005f5e1000401e9b0e77ef8dd743e41bc09be82d2d58887772dca08cfc55865f9e1c3e9f98a7b0100000002540be4000017a914f807ce1c9b15116c73dfd05e4289b8fcbfdbe0268701a8c51a66d8adefd225a2480cbe76009c262a19ca324c662e6c96a338da65e26a010000000005f5e1000017a914f807ce1c9b15116c73dfd05e4289b8fcbfdbe026870125b251070e29ca19043cf33ccd7324e2ddab03ecc4ae0b5e77c4fc0e5cf6c95a010000000005f5d354001976a9147c04bb83d1014839dbda929cdbf8e8711e8eee2c88ac0125b251070e29ca19043cf33ccd7324e2ddab03ecc4ae0b5e77c4fc0e5cf6c95a010000000000000dac000000000000`

Any idea about that @altafan ?

Reply:
that's unexpected.. I was able replicate this with go-elements and with the elements-cli. It turned out that the transaciton is not wrong, but the issuance cannot be owned by a legacy script apparently, or at least by a P2PKH one. Try to use a p2wpkh or a p2sh-p2wpkh instead.

Reply:
I could be able to replicate this using both go-elements and the elements CLI.
It turned out that the transaction is not wrong, but apparently the issuance can be set only for witness inputs (thus those owned by segwit and native-segwit scripts). I suggest you to try using a `p2wpkh` for this test; it should pass as expected (maybe some fix on the transaction's hashForWitnessV0 method will be required)